jdi-light/src/main/java/com/epam/jdi/light/actions/ActionObject.java

```diff
@@ -14,38 +16,42 @@
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.UUID;
 
 import static com.epam.jdi.light.actions.ActionHelper.*;
 import static com.epam.jdi.light.actions.ActionOverride.getOverrideAction;
 import static com.epam.jdi.light.settings.JDISettings.TIMEOUTS;
+import static com.epam.jdi.tools.LinqUtils.map;
 import static com.epam.jdi.tools.ReflectionUtils.isInterface;
-import static java.util.UUID.randomUUID;
 
 public class ActionObject {
     private JoinPoint jp;
     private ProceedingJoinPoint pjp;
-    private UUID uuid;
+    private String processor;
 
-    public ActionObject(JoinPoint joinPoint) {
+    private ActionObject(String name) {
+        this.processor = name;
+    }
+    public ActionObject(JoinPoint joinPoint, String name) {
+        this(name);
         this.jp = joinPoint;
-        baseInit();
     }
-    public ActionObject(ProceedingJoinPoint joinPoint) {
+    public ActionObject(ProceedingJoinPoint joinPoint, String name) {
+        this(name);
         this.pjp = joinPoint;
-        baseInit();
     }
-    private void baseInit() {
-        uuid = randomUUID();
+    private int elementTimeout = -1;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6397461)